app/scripts/angular-apimock.js

```diff
@@ -342,18 +358,13 @@ angular.module('apiMock', [])
 				case 'respond':
 					return httpStatusResponse(command.value);
 				case 'ignore':
-					return req;
-
+				/* falls through */
```

Comment:
The `/* falls through */` might be a bit obscure though. It's not a regular comment. It's a JSHint thing: see "Switch statements" on http://jshint.com/docs/